frame/contracts/src/exec.rs

```diff
@@ -315,9 +315,9 @@ pub struct Stack<'a, T: Config, E> {
 	timestamp: MomentOf<T>,
 	/// The block number at the time of call stack instantiation.
 	block_number: T::BlockNumber,
-	/// The account counter is cached here when accessed. It is written back when the call stack
+	/// The trie_seed is cached here when accessed. It is written back when the call stack
```

Comment:
This comment doesn't really describe what this is, can you add a bit more context here?

Reply:
The explanation of the trie seed is on the `AccountCounter` storage item. I can add a link here. Don't want to repeat the docs here.

Reply:
Added the link.